[Fixes #39] Fix styling on admin and editor pages

bluepichu opened the pull request:
Should fix the remaining points in #39.

bluepichu committed:
[Fixes #39] Fix styling on admin and editor pages

el1t commented:
![Weird issue](https://cloud.githubusercontent.com/assets/5885378/16476622/32b63d5c-3e57-11e6-9c36-ae63c481eeed.png)

bluepichu commented:
I don't really think you can blame me for a virtual keyboard that takes up more or less the entire screen -.-

Suggestions on how to fix?

el1t commented:
It would help if I could scroll until the chrome toolbar up top snaps offscreen, but I don't think there's a way to detect that in the page...

el1t commented:
I bet it can be fixed if the header is not fixed on mobile (no scrolling anyways, so it *shouldn't* be an issue normally), which allows the rest of the page to scroll properly instead of text appearing above the toolbar

bluepichu commented:
Also, I can't really test this issue because it already works nearly-perfectly on Chrome iOS.  (Minor tweak for disabling zoom in landscape mode, maybe, but it works like a charm in portrait.)

zwade commented:
Actually I think that's an artifact of Ellis' build. I don't think that's how stock android behaves

bluepichu committed:
Disable zoom on admin pages

bluepichu commented:
Sorry about the minor history rewrite - a rebase wasn't merging properly and it was the easiest way around it.

bluepichu commented:
@zwade or @el1t can you confirm behavior with stock Android Chrome?

zwade commented:
nm, I was wrong. apparently chrome treats text inputs different from textViews. Either way, I don't think landscape is the way to go on mobile

el1t commented:
Setting header's display to `initial` instead of `fixed` like mentioned above (and removing some margin-top from main) will allow keyboard on the editor screen, but will obviously break the Posts screen

bluepichu commented:
@el1t feel free to fix this.  (You should just be able to push to the branch to update the PR.)  I can't really do it myself since I have no device on which to test.

Also, I think you mean `static` instead of `initial`.

Commit:
Static header on mobile editor page

Also keep editor from getting too small on mobile, since height is usually fixed.

bluepichu commented on client/hbs/editor.hbs:
```diff
@@ -2,7 +2,7 @@
 	{{set this "admin" true}}
 	{{> head.hbs}}
-	<body>
+	<body class="static">
```
I'd instead add class `.editor` to the `<html>` tag and style based off of that.

el1t replied:
whoops only read half the comment my b

Commit:
Rename body class to editor

Commit:
Merge branch 'master' of https://github.com/elmzteam/downtothewire into iss39

# Conflicts:
#	client/hbs/editor.hbs
#	client/hbs/head.hbs

Commit:
Address other half of comment

:|

bluepichu commented:
Sorry for the ridiculous delay.  LGTM, leaving the merge up to you, @el1t.
